plugins/uefi-capsule/fu-acpi-uefi.c

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (C) 2023 Mario Limonciello <mario.limonciello@amd.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#include "config.h"
+
+#include <fwupdplugin.h>
+
+#include <string.h>
+
+#include "fu-acpi-uefi.h"
+
+#define UEFI_HEADER_SIZE	 0x36
+#define FU_EFI_INSYDE_GUID	 "9d4bf935-a674-4710-ba02-bf0aa1758c7b"
+#define INSYDE_QUIRK_COD_WORKING 0x1
+
+struct _FuAcpiUefiTable {
+	FuFirmware parent_instance;
+	guint32 insyde_cod_status;
+	gboolean is_insyde;
+	gchar *creator_id;
+	gchar *oem_id;
+	gchar *guid;
+};
+
+struct FuAcpiInsydeQuirkSection {
+	gchar signature[6];
+	guint32 size;
+	guint32 flags;
```

Comment:
maybe add a `\* le *\` comment for our futureselves? Also, I think you need to sprinkle some GUINT32_FROM_LE's around.

Reply:
I added this.